Makefile

```diff
@@ -265,6 +265,7 @@ QEMU_OPTS_COMMON= -m $(QEMU_MEMORY) -smp 4  $(QEMU_OPTS_BIOS) \
         -serial mon:stdio      \
         -global ICH9-LPC.noreboot=false -watchdog-action reset \
         -rtc base=utc,clock=rt \
+		-device virtio-rng-pci \
```

Comment:
This will add the virtio RNG device for all cases, but the issue with iPXE was happening only when running without KVM support (NOACCEL=1), wouldn't be better to add it only for NOACCEL cases?

Also, please, fix the indentation (use spaces instead of TAB)....

Reply:
We can start with that, sure. If we hit the problem later with acceleration, we can move it up there.

Reply:
Done